site/content/en/docs/tasks/run_flux.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Run A Flux Minicluster"
+date: 2022-02-14
+weight: 6
+description: >
+  Run a Kueue scheduled Flux minicluster.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Flux Operator's](https://flux-framework.org/flux-operator/) Mini Clusters.
+
+For more information on Flux Operator visit [Flux Operator](https://flux-framework.org/flux-operator/).
+
+
+## Before you begin
+
+Please check [Run A Job](/docs/tasks/run_jobs) for details on the initial cluster setup and queue configuration.
+
+## 1. MiniCluster definition
+
+Since a Flux MiniCluster uses  a `batch/job` to run the actual workload, no extra components are needed for it to be managed by Kueue, However the following aspects should be taken into account:
+
+### a. Queue selection
+
+The target local queue should be specified in the `spec.jobLabels` section of the minicluster configuration.
+
+```yaml
+  jobLabels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+### b. Configure the resource needs
+
+The resource needs of the workload can be configured in the `spec.container[].resources` sections of the minicluster configuration.
```

Comment:
```suggestion
The resource needs of the workload can be configured in the `spec.container[*].resources` sections of the minicluster configuration.
```

It would be good to do the same as other places.
example: https://kueue.sigs.k8s.io/docs/tasks/administer_cluster_quotas/#2-create-a-clusterqueue-referencing-the-flavors

Reply:
That might be hard to keep up to date / synced with our docs - so another suggestion is to just link to our documentation https://flux-framework.org/flux-operator/getting_started/custom-resource-definition.html#id3

Reply:
My suggestion is to add `*` to the array :)

Reply:
This means I didn't suggest referring to other documentation :)

Reply:
Or sorry, it was hard to get the full context when you just said "do the same as other places" with a link. I did see the suggestion. And I think it still might be good to refer to the docs on our site!